modules/datastore/modules/datastore_mysql_import/README.md

```diff
@@ -1,12 +1,8 @@
-# DKAN Datastore MySQL Import
+# DKAN Datastore MySQL Import {#mysqlimport}
 
-This is an **expiremental** module to import CSVs into the DKAN datastore using 
-MySQL's native LOAD DATA function, similar to running a `mysqlimport`
-command. It is over 5000% faster than DKAN's default datastore importer, but
-requires a MySQL database and that the Drupal DB user have permissions to use
+This an **expiremental** module to import CSVs into the DKAN datastore using MySQL's native LOAD DATA function, similar to running a `mysqlimport` command. It is over 5000% faster than DKAN's default datastore importer, but requires a MySQL database and that the %Drupal DB user have permissions to use
```

Comment:
Is this still considered to be experimental? We should also probably mention that it doesn't currently have TSV support.